magstock/templates/regextra.html

```diff
@@ -117,4 +117,29 @@
             <textarea name="coming_with" class="form-control" placeholder="Full legal names of everyone in your group">{{ attendee.coming_with }}</textarea>
         </div>
     </div>
+
+    {% if c.PAGE_PATH == '/registration/form' %}
+        <div class="form-group" id="allergies">
+            <label class="col-sm-2 control-label optional-field">Site Number</label>
+            <div class="col-sm-6">
+                <input type="text" class="form-control" name="site_number" value="{{ attendee.site_number }}" placeholder="Admin-only field" />
+            </div>
+        </div>
+
+        <div class="form-group" id="allergies">
+            <label class="col-sm-2 control-label optional-field">License Plate</label>
+            <div class="col-sm-6">
+                <input type="text" class="form-control" name="license_plate" value="{{ attendee.license_plate }}" placeholder="Admin-only field" />
+            </div>
+        </div>
+    {% endif %}
 </div>
+
```

Comment:
This is the block that turns off the prereg form using Javascript.  We might want to make this configurable in the core Uber plugin in the long run, but for now this works.